sympy/sets/sets.py

```diff
@@ -1877,10 +1877,10 @@ def _union(self, other):
 
     def _contains(self, other):
         """
-        Tests whether an element, other, is in the set.
+        Tests whether an element 'other' is in the set.
```

Comment:
This was correct prior to the change. Example "My sister, Sally, was here yesterday."